toucan_connectors/one_drive/one_drive_connector.py

```diff
@@ -52,8 +60,18 @@ class OneDriveDataSource(ToucanDataSource):
     )
 
 
-class OneDriveConnector(ToucanConnector):
+def _prepare_workbook_elements(data_source):
+    if data_source.sheet:
+        workbook_elements_list = data_source.sheet
+        workbook_key_column = '__sheetname__'
+    else:
+        workbook_elements_list = data_source.table
+        workbook_key_column = '__tablename__'
+    workbook_elements_list = workbook_elements_list.replace(', ', ',').split(',')
```

Comment:
```suggestion
    workbook_elements_list = [a.strip() for a in workbook_elements_list.split(",")]
```
because your line doesn't handle `' ,'`

Reply:
:+1: